packages/babel-types/src/traverse/traverseFast.ts

```diff
@@ -1,14 +1,14 @@
-// @flow
 import { VISITOR_KEYS } from "../definitions";
+import type * as t from "../types";
 
 /**
  * A prefix AST traversal implementation meant for simple searching
  * and processing.
  */
 export default function traverseFast(
-  node: Object,
-  enter: (node: BabelNode, opts?: Object) => void,
-  opts?: Object,
+  node: t.Node | null | undefined,
+  enter: (node: t.Node, opts?: any) => void,
+  opts?: any,
 ): void {
```

Comment:
We could parametrize `opts` to `T` rather than `any`, so that the type is "forwarded" to the callback.

Reply:
added todo - it does not really align with default value for it being empty object